tinlake-ui/containers/DataQuery/fortunaFiDailyInvestorBalances.ts

```diff
@@ -0,0 +1,88 @@
+import gql from 'graphql-tag'
+import Apollo from '../../services/apollo'
+import { downloadCSV } from '../../utils/export'
+import { csvName } from './queries'
+
+interface IDaily {
+  account: {
+    id: string
+  }
+  day: {
+    id: string
+  }
+  seniorTokenValue: string
+  juniorTokenValue: string
+}
+
+interface IResponse {
+  dailyInvestorTokenBalances: IDaily[]
+  _meta: {
+    block: {
+      hash: string
+    }
+  }
+}
+
+const fetch = async (skip: number, first: number, blockHash: string | null): Promise<IResponse> => {
+  return await Apollo.runCustomQuery(gql`
+      {
+        dailyInvestorTokenBalances(where:{pool:"0x4B6CA198d257D755A5275648D471FE09931b764A"}, first: ${first}, skip: ${skip} ${
+    blockHash ? `, block: { hash: "${blockHash}" }` : ''
+  }) {
+          account{
+            id
+          }
+          day {
+            id
+          }
+          seniorTokenValue
+          juniorTokenValue
+        }
+        _meta {
+          block {
+            hash
+            number
+          }
+        }
+      }
+    `)
+}
+
+export async function fortunaFiDailyInvestorBalances() {
+  let start = 0
+  let limit = 1000
```

Comment:
To fix the linter warning:
```suggestion
  const limit = 1000
```